dependency-check-suppressions.xml

```diff
@@ -112,4 +112,31 @@
     <packageUrl regex="true">^pkg:maven/io\.netty/netty.*@2\.0\.46\.Final$</packageUrl>
     <cve>CVE-2022-24823</cve>
     </suppress>
+    <suppress>
```

Comment:
This is not the correct format on suppressions but I believe someone else is already fixing the issues so please remove this when it comes to time to merge its fine for now to get build to pass